cmd/config.go

```diff
@@ -22,25 +22,31 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/pilosa/pilosa/ctl"
+	"github.com/pilosa/pilosa/server"
 )
 
 var Conf *ctl.ConfigCommand
 
 func NewConfigCommand(stdin io.Reader, stdout, stderr io.Writer) *cobra.Command {
 	Conf = ctl.NewConfigCommand(os.Stdin, os.Stdout, os.Stderr)
+	Server := server.NewCommand(stdin, stdout, stderr)
```

Comment:
It doesn't show current config as I ran `pilosa config`, correct me if I'm wrong but does this line always create new server with default config, so Server.Config isn't updated with current config.